lib/assessment.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(0) { |memo, num| memo + num }
```

Comment:
Correct. This could have been written even more shortly if you used the `&` trick:
`enum.reduce(&:+)`